### PR TITLE
Highlight today in date picker

### LINKED ISSUE #384: "Today" should be highlighted in Granite.Widgets.DatePicker
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Feature

The Today's date should be highlighted in `Granite.Widgets.DatePicker`.

**Is your feature request related to a problem? Please describe.**

Selecting a date in `Granite.Widgets.DatePicker` feels a bit wonky to me. I feel a bit lost during date selection because I'm always unsure if I selected the right date. My best guess is, that this happens due to the fact that there is no indication of which date is "Today":

Calendar

If you compare this to the calendar view on top of the screen (the one shown when you click on the current date/time), there you can see that Today is highlighted as bold, colored text:

Today Calendar

### PR DESCRIPTION
**Pull request by sebastianlay**
Fixes #384 and improves the styling of the date picker a bit.

Before:
before

After:
after

### CODE REVIEW

**Review by meisenzahl — APPROVED** [review]

**Review by danrabbit — CHANGES_REQUESTED** [review]
We shouldn't be embedding CSS here

**Comment by danrabbit** [comment]
I've proposed styles for Gtk.Calendar here https://github.com/elementary/stylesheet/pull/878

**Comment by cassidyjames** [comment]
So this is superseded by https://github.com/elementary/stylesheet/pull/878?

**Comment by danrabbit** [comment]
@cassidyjames not completely. The part about marking Today is still relevant, but the styles should be removed

File: README.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ To build granite, use `ninja`:
 
     ninja
 
-To install, use `ninja install`
+To install, use `sudo ninja install`
 
-    ninja install
+    sudo ninja install
```
Comment (danrabbit) [inline README.md:33]: This is intentional since ninja will use polkit. Please revert this

File: lib/Widgets/DatePicker.vala [hunk 2]
```diff
@@ -51,8 +51,18 @@ namespace Granite.Widgets {
...
         private bool proc_next_day_selected = true;
 
+        private const string CSS = """
+            calendar
+            {
+                background: transparent;
+                padding: 3px;
+            }
+        """;
+
```
Comment (danrabbit) [inline lib/Widgets/DatePicker.vala:65]: Styles are accounted for in the system stylesheet already, so we shouldn't have them here

File: lib/Widgets/DatePicker.vala [hunk 3]
```diff
@@ -158,5 +181,14 @@ namespace Granite.Widgets {
         private void hide_dropdown () {
             popover.hide ();
         }
+
+        private void on_calendar_month_changed () {
+            if (calendar.year == today.get_year () && calendar.month == today.get_month () - 1) {
+                calendar.mark_day (today.get_day_of_month ());
+            }
+            else {
```
Comment (danrabbit) [inline lib/Widgets/DatePicker.vala:189]: elementary code style is cuddled else
```suggestion
            } else {
```

**Review by danrabbit — CHANGES_REQUESTED** [review]

**Comment by danrabbit** [comment]
Gonna close this since there's been no response to code review for over a month. Please feel free to reopen if changes are made :)

**Pull request closed without merging**